### ISSUE TITLE
The problems when running the code

### ISSUE THREAD

**Issue by Castallyf** [datamllab/Policy-GNN#1]
Hello, I have read your paper on KDD2020 and the code recently, it is an interesting work and receive great performance. Congratulations!
But I have some problems about the code. 
First, in your train.py, your code just output the best test accuracy among all the test result but not the test accuracy under the best performance of validation sets, which is not consistent with the Section 4.1 in your paper, "The proposed model and baselines are all trained and evaluated with the complete graph structure and node features in the training dataset, without using the node labels in the held-out validation and testing sets. The model hyper-parameters are selected based on the performance on the validation set.".  Could you please tell me if I misunderstand the idea of your paper? Or is there something wrong in your code?
The second question is that I have a poor performance on the datasets "Citeseer" and "Pubmed", so I wonder have you change some parameters when running the models on these two datasets? 
Looking forward to your kindly reply.
Thank you so much!

**Comment by lhenry15** [comment]
Hi, if you look into the line 34, it states that we perform evaluation on testing when validation dataset is achieving a higher performance. And the line 36 is to record the best test accuracy we have gained for now. So, what we are doing is testing the meta-policy with GNN whenever the meta-policy gains the improvement on validation dataset.  As for the hyper-parameter of the other two dataset, I will release them once I finish the repository (still working on re-factoring the experiments)

**Comment by lhenry15** [comment]
Hi, if you look into the line 34, it states that we perform evaluation on testing when validation dataset is achieving a higher performance. And the line 36 is to record the best test accuracy we have gained for now. So, what we are doing is testing the meta-policy with GNN whenever the meta-policy gains the improvement on validation dataset.

**Comment by lhenry15** [comment]
As for the hyper-parameter of the 'Citeseer' and 'Pubmed', I will provide them in the train.py after I fix the bugs of the current refactored code.

**Comment by Castallyf** [comment]
I got it! Thank you very much!
